test/functional/config.js

```diff
@@ -92,10 +92,6 @@ export default async function({ readConfigFile }) {
         pathname: '/app/kibana',
         hash: '/dev_tools/console',
       },
-      account: {
```

Comment:
__note:__ I haven't found any place that would use this, so just removed instead of updating.